frontend/src/app/modules/home/projects/edit/edit.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="title"><button routerLink="../" tooltipPosition="bottom" pTooltip="Back to Projects" pButton pRipple
+    icon="pi pi-chevron-left " class="p-button-outlined back-button"></button>{{project?.name}}
+<div class="header-buttons" >
+    <div class="buttons-container">
+        <button #saveBut pButton pRipple class="p-button-primary p-button-outlined" label="Save" (click)="updateProjectFunction()"></button>
+        <button pButton pRipple class="p-button-danger p-button-outlined" label="Remove" (click)="deleteProject(this.id)"></button>
```

Comment:
Why do you use "this"?

Reply:
this.id is the project id which is defined via activatedRoute

Reply:
If this is not a dynamic value you can set it in inside the method
And I'm believe that you need "this" here

Reply:
Fix